vital/iterator.h

```diff
@@ -200,8 +212,8 @@ class base_iterator
    */
   base_iterator()
     : m_next_value_func( nullptr )
-    , m_cur_val_ptr( nullptr )
-    , m_at_end( true )
+      , m_cur_val_ptr( nullptr )
+      , m_at_end( true )
```

Comment:
This is probably wrong... I suspect either a misconfiguration or unimplemented feature.